stdlib/package.json

```diff
@@ -40,6 +40,6 @@
   "dependencies": {},
   "scripts": {
     "build": "node --harmony-top-level-await ./build.mjs",
-    "clean": "del-cli '**/*.wasm' '**/*.wasm.modsig' '**/*.wasm.wast' '!stdlib-external/**'"
+    "clean": "del-cli '**/*.wasm' '**/*.wasm.*' '**/*.gr.mashtree' '!stdlib-external/**'"
```

Comment:
Oh wow, I hadn't realized we were even generating `.mashtree` files. We probably don't need to generate those when running the tests.

Reply:
We don't by default, but I did something to create them and then I couldn't automatically clean them up.